environments/production.yml

```diff
@@ -4,6 +4,7 @@ service:
 env:
   PORT: 3000
   SNOOTY_DB_NAME: snooty_prod
+  SNOOTY_PROD_DB_NAME: snooty_dotcomprd
   POOL_DB_NAME: pool
   SNOOTY_ENV: dotcomprd
```

Comment:
Forgot to mention this in previous PRs, but can we update the `sample.env` file with the new envs and suitable default (or empty) values? Might be good for setup and documentation purposes

Reply:
good catch! done